sdk/appconfiguration/Azure.Data.AppConfiguration/tests/samples/Sample2_HelloWorldExtended.cs

```diff
@@ -47,13 +44,23 @@ public async Task HelloWorldExtended()
             #region Snippet:AzConfigSample2_GetConfigurationSettingsAsync
             var selector = new SettingSelector { LabelFilter = "production" };
 
-            Debug.WriteLine("Settings for Production environment:");
+            Console.WriteLine("Settings for Production environment:");
```

Comment:
![image](https://github.com/Azure/azure-sdk-for-net/assets/913445/03b6766f-f41e-478b-9b68-3c087408c9dc)